src/plugins/plugin-grpc.js

```diff
@@ -442,6 +470,12 @@ module.exports = [
     unpatch: unpatchClient
   },
   {
+    file: 'src/node/src/metadata.js',
+    versions: SUPPORTED_VERSIONS,
+    patch: patchMetadata,
```

Comment:
Could you also add a small comment here (similar to above) stating why a no-op unpatching is safe (nothing is modified when we grab the constructor)?